util/flags.py

```diff
@@ -51,9 +51,11 @@ def create_flags():
 
     f.DEFINE_integer('export_batch_size', 1, 'number of elements per batch on the exported graph')
 
-    # Performance(UNSUPPORTED)
-    f.DEFINE_integer('inter_op_parallelism_threads', 0, 'number of inter-op parallelism threads - see tf.ConfigProto for more details')
-    f.DEFINE_integer('intra_op_parallelism_threads', 0, 'number of intra-op parallelism threads - see tf.ConfigProto for more details')
+    # Performance
+
+    f.DEFINE_integer('inter_op_parallelism_threads', 0, 'number of inter-op parallelism threads - see tf.ConfigProto for more details. USE OF THIS FLAG IS UNSUPPORTED')
+    f.DEFINE_integer('intra_op_parallelism_threads', 0, 'number of intra-op parallelism threads - see tf.ConfigProto for more details. USE OF THIS FLAG IS UNSUPPORTED')
+    f.DEFINE_boolean('use_cudnn_rnn', False, 'use CuDNN RNN backend for training on GPU')
```

Comment:
Maybe worth documenting that this will produce a checkpoint that cannot be trained on anything else than CUDA.